src/main.rs

```diff
@@ -94,26 +94,27 @@ fn main() -> io::Result<()> {
     let mut terminal = Terminal::new(CrosstermBackend::new(io::stdout()))?;
     terminal.hide_cursor()?;
     let num_reqs = args.num_requests.get();
-    let sp = Spinner::new(Spinners::Dots9, format!("Sending {} requests", num_reqs));
+    let sp =
```

Comment:
This your more struct line length style?

Reply:
whoops, didn't see that this got in there, sorry. I think it's because I have a `.rustfmt.toml` file in my home directory that sets `max_width = 80`.

Reply:
No worries.